specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -507,6 +507,53 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/quotaType/{quotaType}/usages": {
+      "get": {
+        "summary": "Gets subscription core usages for location, for a given type of quota.",
+        "description": "Gets the usage in cores for a subscription in a given location, for a specific quota type.",
+        "operationId": "GetUsagesInLocation_list",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
+          {
+            "name": "quotaType",
+            "description": "The type of quota to get usages for",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/CsmUsageQuotaCollection"
```

Comment:
You shouldn't be using the obsolete acronym CSM (the old internal name for ARM). Just call it 'UsageQuotaCollection'

Reply:
[If this is a breaking change with GA APIs it may be simpler not to fix it though.]

Reply:
There are other APIs that reference this definition, it will be simpler not to change

Reply:
Okay, I guess we are stuck with it then. ;(